servatrice/servatrice.ini.example

```diff
@@ -107,6 +107,14 @@ allowedpunctuation=_.-
 ; If a username can begin with punctuation defined in allowedpunctuation
 allowpunctuationprefix=false
 
+; Disallow usernames containing these words. This list is comma seperated
+disallowedwords="admin"
```

Comment:
I'd add a clearer example of the meaning of "containing", eg. that "ass" matches "grasshopper"